client/dashboard/sites/hosting-feature-activation-modal/error-content-info.tsx

```diff
@@ -0,0 +1,206 @@
+		description: __(
+			'Your domain is not properly set up to point to your site. Reset your domain’s A records in the Domains section to fix this.'
+		),
+		// eslint-disable-next-line wpcalypso/i18n-unlocalized-url
+		supportUrl: 'https://wordpress.com/support/move-domain/setting-custom-a-records/',
+	},
+	[ EligibilityErrors.EMAIL_UNVERIFIED ]: {
+		title: __( 'Confirm your email address' ),
+		description: __(
+			'Check your email for a message we sent you when you signed up. Click the link inside to confirm your email address. You may have to check your email client’s spam folder.'
+		),
+	},
+	[ EligibilityErrors.EXCESSIVE_DISK_SPACE ]: {
+		// TODO: Implement upsells
+		title: __( 'Increase storage space' ),
+		description: __(
+			'Your site does not have enough available storage space. Please purchase a plan with additional storage or contact our support team for help.'
+		),
+	},
+	[ EligibilityErrors.IS_STAGING_SITE ]: {
+		title: __( 'Create a new staging site' ),
+		description: __(
+			'Hosting features cannot be activated for a staging site. Create a new staging site to continue.'
+		),
+	},
+};
+
+/*
+	For Atomic sites on plans below Business it will return the holds TRANSFER_ALREADY_EXISTS and NO_BUSINESS_PLAN.
+	Because TRANSFER_ALREADY_EXISTS is present and 'blocking' it will show an "Upload in progress" notice even when there isn't one.
+	In this scenario we need to check if it's an Atomic ste (TRANSFER_ALREADY_EXISTS) on a plan below Business (NO_BUSINESS_PLAN)
+	so we can stop the render of "Upload in progress" and prompt them to upgrade instead.
+*/
+function isAtomicSiteWithoutBusinessPlan( errors: AutomatedTransferEligibilityError[] ) {
+	return [ EligibilityErrors.TRANSFER_ALREADY_EXISTS, EligibilityErrors.NO_BUSINESS_PLAN ].every(
+		( code ) => errors.some( ( error ) => error.code === code )
+	);
+}
+
+function findFirstBlockingError( errors: AutomatedTransferEligibilityError[] ) {
+	const error = errors.find( ( err ) => err.code in BlockingMessages );
+	if ( ! error ) {
+		return null;
+	}
+
+	return BlockingMessages[ error.code as EligibilityErrors ];
+}
+
+function findHoldingErrors( errors: AutomatedTransferEligibilityError[] ) {
+	return errors.reduce( ( acc, err ) => {
+		const holdingMessage = HoldingMessages[ err.code as EligibilityErrors ];
+		if ( holdingMessage ) {
+			acc.push( holdingMessage );
+		}
+
+		return acc;
+	}, [] as HoldingMessage[] );
+}
+
+export function hasBlockingError( errors: AutomatedTransferEligibilityError[] ) {
+	return findFirstBlockingError( errors ) !== null;
+}
+
+export function ErrorContentInfo( { errors }: { errors: AutomatedTransferEligibilityError[] } ) {
+	const blocking = ! isAtomicSiteWithoutBusinessPlan( errors ) && findFirstBlockingError( errors );
+	const holds = findHoldingErrors( errors );
+
+	return (
+		<VStack>
+			{ blocking && (
+				<Notice variant={ blocking.variant }>
+					{ blocking.message }
+					{ blocking.supportUrl && (
+						<>
+							{ ' ' }
```

Comment:
Is this whitespace intended? (Also in several other places in this PR)

Reply:
Yes, is to have a space between the text (message), and the "Learn more" link. Open to other approaches!